src/utils/GridWrapper.js

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+const GridWrapper = ({ children, cols }) => {
```

Comment:
Should we go ahead and add a `rows` prop as well and default both to `1`?

Reply:
I was thinking about that. We should add rows as an option, but default it to auto. Often times with grid, you don't need to set a finite number of rows, and if we can leave it as an optional component I think that would be nice.

Reply:
Sounds good to me.